### DISCUSSION TITLE
fio seq write performance drop inquiry in kernel v6.6

### DISCUSSION THREAD

**Discussion by jinsu-1** [axboe/fio#1835] [Q&A] [answered]
Is there a reason why the fio seq write performance of 6.6 kernel is poor when the same fio seq write command is executed in v6.1 kernel and v6.6 kernel?

**Comment by axboe** [comment, marked as answer]
First of all, there are zero details here, so nobody is going to be able to render any qualified opinions on this. Secondly, this sounds like something that isn't a fio issue and should be reported elsewhere.
